guides/Dataset.md

```diff
@@ -391,7 +369,7 @@ NOTE: If you have a DOI, the citation text can be [automatically generated](http
 `schema:sameAs` is used here for the following reasons:
 
 1. It doesn't add too many more statements that might increase the page weight (which may impact major search engine crawlers stopping the crawl of schema.org markup).
-2. Crawlers that follow the URL for the short DOI can retrieve structured metadata for the DOI itself: 
+2. Crawlers that follow the URL for the short DOI can retrieve structured metadata for the DOI itself:
```

Comment:
It should be explained that such an operation does not retrieve the JSON-LD offered by the content source. Instead an alternative, less informative, and potentially out of date version is presented from CrossRef.

Reply:
Agreed. This section isn't the subject of this PR, which is focused on context retrieval, but I agree with your summary.